app/assets/javascripts/teams.js

```diff
@@ -0,0 +1,44 @@
+$(document).ready(function() {
+  $(".dropdown-button").dropdown();
+
+  loadLinks();
+
+  function loadLinks() {
+    $.ajax({
+      method: "GET",
+      url: "/api/v1/teams",
+      dataType: "JSON",
+      success: function(response){
```

Comment:
Refactoring the success function here might make it more clear what is happening to the data that you're getting from the AJAX call.